File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/riyousha/SaveRiyoushaAdminCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;


/**
 * 運営者ユーザー格納Dto
 */
public class SaveRiyoushaAdminCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 利用者運営者Dto */
    private RiyoushaAdminDto riyoushaAdminDto = new RiyoushaAdminDto();

    /**
     * 利用者運営者Dtoを取得する
     *
     * @return 利用者運営者Dto
     */
    public RiyoushaAdminDto getRiyoushaAdminDto() {
        return riyoushaAdminDto;
    }

    /**
     * 利用者運営者Dtoを設定する
     *
     * @param riyoushaAdminDto 利用者運営者Dto
     */
    public void setRiyoushaAdminDto(final RiyoushaAdminDto riyoushaAdminDto) {
        this.riyoushaAdminDto = riyoushaAdminDto;
    }

}

File: app/front/src/components/main/dto/riyousha/saveRiyoushaAdminCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { RiyoushaAdminDto, type RiyoushaAdminDtoInterface } from "./riyoushaAdminDto";

interface SaveRiyoushaAdminCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 利用者運営者Dto */
    riyoushaAdminDto: RiyoushaAdminDtoInterface;

}


class SaveRiyoushaAdminCapsuleDto extends FrameworkCapsuleDto
    implements SaveRiyoushaAdminCapsuleDtoInterface {

    /** 利用者運営者Dto */
    riyoushaAdminDto: RiyoushaAdminDtoInterface;

    constructor() {
        super();

        this.riyoushaAdminDto = new RiyoushaAdminDto();
    }
}

export { type SaveRiyoushaAdminCapsuleDtoInterface, SaveRiyoushaAdminCapsuleDto }
